tests/test_buildrunner_files.py

```diff
@@ -48,7 +48,8 @@ def _test_buildrunner_file(test_dir, file_name, args, exit_code):
         command_line = [
             'buildrunner-test',
             '-d', top_dir_path,
-            '-b', temp_dir,
+            # FIXME containerize step looks for buildrunner.results/artifacts/ and fails with temp_dir
+            # '-b', temp_dir,
```

Comment:
This doesn't alter existing test behavior since previous to this PR the test_runner was not using the build_results_dir pasted in here. I am creating a ticket to fix this.